internal/cmd/benchmark/go.mod

```
@@ -0,0 +1,31 @@
+module go.mongodb.go/mongo-driver/internal/cmd/benchmark
+
+go 1.22
+
+toolchain go1.22.3
```

Do we need to use Go 1.22 for anything specific? If not, I recommend removing the `toolchain` directive and specifying `go 1.20` to match the Go version we run in CI currently.

```suggestion
go 1.20
```

---

I think this automatically updated from go mod tidy, but 1.22 isn't used for anything specific. Thank you!